substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/Target_java_lang_Module.java

```diff
@@ -51,9 +51,15 @@ public boolean isNativeAccessEnabled() {
     @TargetElement(onlyWith = JDK21OrEarlier.class)
     public native void ensureNativeAccess(Class<?> owner, String methodName);
 
-    @Alias
+    /**
+     * Allow all native access. This is a workaround for JDK-8331671 until we have a better
+     * solution. (GR-57608)
+     */
```

Comment:
fyi @zakkak @jerboaa

Reply:
@zapster OK. Thanks for the heads-up. I'd have thought `--enable-native-access=<graal-module>` to be added where it needs to be. Like all the `--add-opens/add-exports`. Is that the longer-term plan?

Reply:
yes, we do that already. see the changes in the driver code in this pr. the problem with this method is that it unconditionally pulls in all the error handling code. to give you an idea, this results in doubling the number of reachable types in simple `mx helloworld`. the planned solution is to make sure that all the code paths leading here are force inlined so that the module is constant and we can fold the error/warning branch away.

Reply:
I see.